.github/workflows/samples.yml

```diff
@@ -61,7 +61,6 @@ jobs:
 
   web:
     runs-on: ubuntu-20.04
-    container: gcr.io/iree-oss/emscripten@sha256:2dd4c52f1bb499ab365aad0111fe5538b685d88af38636b409b0cf6a576ab214
     env:
       VENV_DIR: ${{ github.workspace }}/.venv
```

Comment:
might be cleaner to add env variables here in future btw

Reply:
Trueee... I could simplify the various places this is done, like https://github.com/iree-org/iree/blob/3bdd90912ddd5783dd4346dfec372f0d3107f7bd/.github/workflows/pkgci_test_android.yml#L64-L69

Reply:
Do we prefer one over the other? Most workflows I've looked in seem to follow https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-environment-variables-for-a-single-workflow and not https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-an-environment-variable but I don't know what we should prefere.